accounting/accounting.go

```diff
@@ -97,6 +97,28 @@ func bytesAreZero(b []byte) bool {
 	return true
 }
 
+func (accounting *State) closeAccount(addr types.Address) {
+	if accounting.AlgoUpdates == nil {
```

Comment:
This should be impossible because something earlier in the payment txn must have set it

Reply:
I agree, but prefer to keep all the edge cases handled. Maybe in the future `Clear` can re-initialize to empty maps.